subworkflows/local/create_align_index/main.nf

```diff
@@ -0,0 +1,67 @@
+include { BOWTIE_BUILD as BOWTIE1_BUILD } from '../../../modules/nf-core/bowtie/build'
+include { BOWTIE2_BUILD                 } from '../../../modules/nf-core/bowtie2/build'
+include { BWAMEM2_INDEX                 } from '../../../modules/nf-core/bwamem2/index'
+include { BWA_INDEX as BWAMEM1_INDEX    } from '../../../modules/nf-core/bwa/index'
+include { DRAGMAP_HASHTABLE             } from '../../../modules/nf-core/dragmap/hashtable'
+
+workflow CREATE_ALIGN_INDEX {
```

Comment:
This doesn't seem to be named very intuitively. e.g. I would expect `CREATE_ALIGN_INDEX` and `CREATE_ALIGN_INDEX_WITH_GFF` to be quite similar, but they're using a completely different set of tools.

It would probably be helpful if the subworkflows followed the nf-core naming conventions (https://nf-co.re/docs/guidelines/components/subworkflows#name-format-of-subworkflow-files), even if they'll stay local.

Reply:
yeah, I totally did not follow any guidelines on that.
For me I was more grouping together tools need a similar set of files.
Hence I didn't want to list the tools in the name of the subworkflow to allow for easier update

Reply:
Yeah, this doesn't make much sense to me for naming. I'll trust that you fix it before any release/ production usage.